.travis.yml

```diff
@@ -1,9 +1,18 @@
 language: ruby
 rvm:
-- 2.3.3
+  - 2.3.3
+
+services:
+  - docker
+
+sudo: required
 
 script: bash ./bin/automated
 
+deploy:
+  - provider: script
+    script: ./bin/deploy
+
```

Comment:
Why do you need an interpreter (`bash`) in one of the scripts and none in the other?